### PR TITLE
chore: Migrate bigtable synth.py to bazel

### PR DESCRIPTION
**Pull request by vam-google**

### CODE REVIEW

**Comment by vam-google** [comment]
@chingor13 PTAL

File: synth.py [hunk 1]
```diff
@@ -70,12 +73,12 @@ def generate_data_api(gapic):
   ]
 
   package_name = f'com.google.bigtable.v2'
-  java.fix_proto_headers(library / f'proto-google-cloud-bigtable-v2')
-  java.fix_grpc_headers(library / f'grpc-google-cloud-bigtable-v2', package_name)
+  java.fix_proto_headers(library / f'proto-google-cloud-bigtable-v2-java')
```
Comment (kolea2) [inline synth.py]: I don't think these should be changing. This should have generated at least the changes found here: https://github.com/googleapis/java-bigtable/pull/222 which we have yet to merge in.

Comment (vam-google) [reply synth.py]: Sorry, that was silly me. I updated the synth.py, but did not update my command to call synthtool, so I just generated java-gerafeas (the repo I was migrating before java-bigtable).

**Comment by vam-google** [comment]
@kolea2 PTAL. I regenerated the files, and now it seems like there are a lot of changes, including breaking ones =(.

**Comment by chingor13** [comment]
As an FYI, we added a `required` option to `s.copy()`/`s.move()` which will cause the command to fail if nothing was copied (as a sanity check)

**Comment by kolea2** [comment]
@vam-google thanks! It looks like the same tests are failing in the other PR, I think we need to update a test. I'll take a look and let you know when this is good to come back to. Sorry for the inconvenience!

**Comment by vam-google** [comment]
@kolea2 Friendly ping

**Comment by vam-google** [comment]
Thanks @kolea2 Can we now merge it? (I don't have right to merge)

**Pull request merged**